packages/shylu/shylu_node/basker/src/shylubasker_order_btf.hpp

```diff
@@ -754,12 +757,14 @@ namespace BaskerNS
       //Int break_size = 500000;
       printf( " > debug: break_size = %d\n",break_size );
       #else
-      double break_size = ceil(total_work_estimate*(
-            ((double)1.0/num_threads) + 
-            ((double)BASKER_BTF_IMBALANCE)));
+      // A block if it is larger than work esitimate assigned to one thread
+      double break_fact = 0.7;
```

Comment:
Is the 0.7 constant determined from experiments and experience, or is there another reason for this value?